spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesInformerDiscoveryClientAutoConfiguration.java

```diff
@@ -66,6 +66,7 @@ public KubernetesDiscoveryClientHealthIndicatorInitializer indicatorInitializer(
 		return new KubernetesDiscoveryClientHealthIndicatorInitializer(podUtils, applicationEventPublisher);
 	}
 
+	@Deprecated(forRemoval = true)
 	@Bean
 	@ConditionalOnMissingBean
 	public KubernetesInformerDiscoveryClient kubernetesInformerDiscoveryClient(
```

Comment:
deprecate this one as we do not use it anymore, because we do not care about the namespace being provided to the discovery client.